frontend/src/components/LoginPrompt.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+
+const LoginPrompt = (props) => {
+  if(props.type==="org"){
```

Comment:
Logic can be optimised using ternary operator.

Reply:
Multiple prompt types will come up in future and hence ternary might become unmanageable in this case.

Reply:
Okay.

Reply:
One thing more, Please try to create PR feature-wise as the smaller the PR, more manageable is code review and easier to track changes later, Identify and fix bugs or issues and even roll back changes in future.

Reply:
Kept on pushing as code review was delayed, since I can't open a new PR from the same branch w/o closing this one. All these changes are from last 2 days...